### PR TITLE
added readme for kotlin/raindrops

### PR DESCRIPTION
**Pull request by uzilan**
Added readme for kotlin/raindrops. Please label also this PR with hacktoberfest. Thanks!

### CODE REVIEW

**Comment by uzilan** [comment]
Weeee! PR no. 100! :)

File: tracks/kotlin/exercises/raindrops/REAMDE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    private val factorsToWords = mapOf(3 to "Pling", 5 to "Plang", 7 to "Plong")
+
+    fun convert(input: Int): String {
+        val words = factorsToWords.filterKeys { input % it == 0 }.values
+
+        return if (words.isEmpty())
+            input.toString()
+        else
+            words.joinToString("")
+
```
Comment (ErikSchierboom) [inline tracks/kotlin/exercises/raindrops/REAMDE.md]: Can this emty line be removed?

File: tracks/kotlin/exercises/raindrops/REAMDE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+### Reasonable Solutions
+```kotlin
+object Raindrops {
+    fun convert(num: Int): String = buildString {
+        if (num % 3 == 0) append("Pling")
+        if (num % 5 == 0) append("Plang")
+        if (num % 7 == 0) append("Plong")
+        if(isEmpty()) append(num)
```
Comment (ErikSchierboom) [inline tracks/kotlin/exercises/raindrops/REAMDE.md]: I think there is a space missing after the `if`.

**Comment by ErikSchierboom** [comment]
Thanks!

**Pull request merged**